sdk/browser/test/helpers/TestmailInbox.ts

```diff
@@ -0,0 +1,59 @@
+import fetch from 'node-fetch'
+import { URL } from 'url'
+import cryptoRandomString from 'crypto-random-string'
+
+const TESTMAIL_API_URL = 'https://api.testmail.app/api/json'
+const TESTMAIL_INBOX_DOMAIN = 'inbox.testmail.app'
+const TESTMAIL_API_KEY = process.env.TESTMAIL_API_KEY
+const TESTMAIL_NAMESPACE = process.env.TESTMAIL_NS
+
+interface InboxOptions {
+  prefix: string
+  suffix: string
+}
+
+interface Email {
+  subject: string
+  body: string
+}
+
+// NOTE: Don't use this helper for mere email generation,
+//       since there are limits for emails sent per month
+//       Use generateEmail() instead
+export class TestmailInbox {
+  private _tag: string
+  private _email: string
+  private _lastEmailTimestamp: number
+
+  constructor({ prefix, suffix }: InboxOptions) {
+    const inboxId = cryptoRandomString({ length: 12 }) // arbitrary length
+    this._tag = `${prefix}.${inboxId}.${suffix}`
+    this._email = `${TESTMAIL_NAMESPACE}.${this._tag}@${TESTMAIL_INBOX_DOMAIN}`
+
+    // if inbox has already been used, ignore its old emails
+    this._lastEmailTimestamp = Date.now() - 60_000
```

Comment:
One minute?

Reply:
And why would we need it at all if inboxId is cryptographically secure random? If you think there are meaningful chances of collision, then we should probably increase the length of `inboxId` instead.

Reply:
We don't need it cryptographically secure random, I just didn't know what else to use in this case.
Simplest solution for me was to use this lib.

One minute is arbitrary. Just in case there will ever be a collision, avoid situation where `livequery` immediately resolves with a very old email.

Reply:
But there should be no collision. And if we think there might be, it's better to increase the `inboxId` length to avoid them.

Reply:
Are there cons of doing it the way I did here?

Reply:
It introduces a magic constant and makes intent harder to follow, ihmo.
But if you feel it's better that way, then let's leave it that way.